lib/CodeGen/CGExpr.cpp

```diff
@@ -2995,6 +3016,10 @@ LValue CodeGenFunction::EmitArraySubscriptExpr(const ArraySubscriptExpr *E,
 
   // TODO: Preserve/extend path TBAA metadata?
 
+  if (CheckBounds) {
+    EmitCheckedCSubscriptCheck(LV, CheckBounds);
```

Comment:
Clang style is to generally avoid single line statements wrapped in braces.  They drop the braces in favor of brevity.